Scenarios/CGCT/CGCT.vsimporter/CGCT-WinStore10/CGCT-Debug-xcvars.txt

```diff
@@ -0,0 +1,115 @@
+CLANG_WARN__DUPLICATE_METHOD_MATCH = YES
+CODESIGNING_FOLDER_PATH = sbuild/Debug-iphoneos/CGCT.app
+CODE_SIGN_IDENTITY[sdk=iphoneos*] = iPhone Developer
+CONFIGURATION = Debug
+CONFIGURATION_BUILD_DIR = sbuild/Debug-iphoneos
+CONFIGURATION_TEMP_DIR = sbuild/CGCT.sbuild/Debug-iphoneos
+COPY_PHASE_STRIP = NO
+CURRENT_ARCH = msvc
+DEAD_CODE_STRIPPING = YES
+DEBUG_INFORMATION_FORMAT = dwarf
+DERIVED_FILES_DIR = sbuild/CGCT.sbuild/Debug-iphoneos/CGCT.sbuild/DerivedSources
+DERIVED_FILE_DIR = sbuild/CGCT.sbuild/Debug-iphoneos/CGCT.sbuild/DerivedSources
+DERIVED_SOURCES_DIR = sbuild/CGCT.sbuild/Debug-iphoneos/CGCT.sbuild/DerivedSources
+EFFECTIVE_PLATFORM_NAME = -iphoneos
+ENABLE_STRICT_OBJC_MSGSEND = YES
+ENABLE_TESTABILITY = YES
+EXCLUDED_RECURSIVE_SEARCH_PATH_SUBDIRECTORIES = *.nib *.lproj *.framework *.gch (*) .DS_Store CVS .svn .git .hg *.xcodeproj *.xcode *.pbproj *.pbxproj *.xcassets
+EXECUTABLE_FOLDER_PATH = CGCT.app
+EXECUTABLE_NAME = CGCT
+EXECUTABLE_PATH = CGCT.app/CGCT
+FRAMEWORK_SEARCH_PATHS = 
+FULL_PRODUCT_NAME = CGCT.app
+GCC_C_LANGUAGE_STANDARD = gnu99
+GCC_DYNAMIC_NO_PIC = NO
+GCC_NO_COMMON_BLOCKS = YES
+GCC_OPTIMIZATION_LEVEL = 0
+GCC_PREFIX_HEADER = 
+GCC_PREPROCESSOR_DEFINITIONS = DEBUG=1
+GCC_REUSE_STRINGS = YES
+GCC_STRICT_ALIASING = YES
+GCC_UNROLL_LOOPS = NO
+GCC_WARN_64_TO_32_BIT_CONVERSION = YES
+GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR
+GCC_WARN_UNDECLARED_SELECTOR = YES
+GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE
+GCC_WARN_UNUSED_FUNCTION = YES
+GCC_WARN_UNUSED_VARIABLE = YES
+HEADERMAP_INCLUDES_FLAT_ENTRIES_FOR_TARGET_BEING_BUILT = YES
+HEADERMAP_INCLUDES_FRAMEWORK_ENTRIES_FOR_ALL_PRODUCT_TYPES = YES
+HEADERMAP_INCLUDES_PROJECT_HEADERS = YES
+HEADER_SEARCH_PATHS = 
+INFOPLIST_FILE = CGCT/Info.plist
+INFOPLIST_PATH = CGCT.app/Info.plist
+IPHONEOS_DEPLOYMENT_TARGET = 10.0
+LD_NO_PIE = NO
+LD_RUNPATH_SEARCH_PATHS = @executable_path/Frameworks
+LIBRARY_SEARCH_PATHS = 
+LINK_FILE_LIST = sbuild/CGCT.sbuild/Debug-iphoneos/CGCT.sbuild/Objects/msvc/CGCT.LinkFileList
+LINK_WITH_STANDARD_LIBRARIES = NO
+LOCALIZED_RESOURCES_FOLDER_PATH = CGCT.app/English.lproj
+MINIMUM_VISUAL_STUDIO_VERSION = 15.0
+MTL_ENABLE_DEBUG_INFO = YES
+OBJECT_FILE_DIR = sbuild/CGCT.sbuild/Debug-iphoneos/CGCT.sbuild/Objects/msvc
+OBJROOT = sbuild
+ONLY_ACTIVE_ARCH = YES
+OTHER_CFLAGS = 
+OTHER_CPLUSPLUSFLAGS = 
+OTHER_LDFLAGS = 
+PLATFORM_NAME = iphoneos
+PLATFORM_TOOLSET = v141
+PRIVATE_HEADERS_FOLDER_PATH = CGCT.app/Private
+PRODUCT_BUNDLE_IDENTIFIER = Microsoft.CGCT
+PRODUCT_NAME = CGCT
+PRODUCT_TYPE = com.apple.product-type.application
+PROJECT = CGCT
+PROJECT_DIR = E:/island/github/henryfox/WinObjC-Samples/Scenarios/CGCT
+PROJECT_FILE_PATH = E:\island\github\henryfox\WinObjC-Samples\Scenarios\CGCT\CGCT.xcodeproj
+PROJECT_NAME = CGCT
+PROJECT_TEMP_DIR = sbuild/CGCT.sbuild
+PROJECT_TEMP_ROOT = sbuild
+PUBLIC_HEADERS_FOLDER_PATH = CGCT.app/Public
+SDKROOT = iphoneos
+SOURCE_ROOT = E:/island/github/henryfox/WinObjC-Samples/Scenarios/CGCT
```

Comment:
You could have a relative path to the sdk root. Specifying the sdk root was [removed](https://github.com/Microsoft/WinObjC/commit/a35fa38ddba2a2acfbdd5d477e8a9fbf7487f961#diff-7601de1d27801d5d35df5638d81ca779) as an [option](https://github.com/Microsoft/WinObjC/commit/a35fa38ddba2a2acfbdd5d477e8a9fbf7487f961#diff-07f94e70dbee2d3e7b6617a4eb802589) and only the [defaults remain](https://github.com/Microsoft/WinObjC/blob/develop/tools/vsimporter/src/BuildSettingsInfo.cpp#L173). The unused defaults should be removed as well.